cypress/e2e/cloud/subscriptions.test.ts

```diff
@@ -432,7 +432,7 @@ describe('Subscriptions', () => {
       .type('t2=//t')
     // add field
     cy.getByTestID('string-parsing-add-rule').click()
-    cy.getByTestID('string-parsing-add-rule-1')
+    cy.getByTestID('string-parsing-add-0')
```

Comment:
accidentally deleted rule here

```suggestion
    cy.getByTestID('string-parsing-add-rule-0')
```